keps/sig-node/0035-20190130-topology-manager.md

```diff
@@ -199,36 +198,43 @@ The Topology Hint currently consists of
 
 The Topology Manager component will be disabled behind a feature gate until graduation from alpha to beta.
 
-#### Policies
-The node-level topology policy takes precedence over the pod-level topology policy.
-The node-level policies will be deprecated later, because the utilization is better for servers to have dynamic policies.
+#### Topology Policy in Pod Spec
+The value of `v1core.PodSpec.Topology.Policy` field indicates the desired topology of Pod's resource assignment.
```

Comment:
if only policy planed right now why not simplify it in following way:
v1core.PodSpec.nodeTopologyPolicy

Reply:
I think if we have a plan to extend Topology (from my point of view just Topology too general, since there are Topology* in PodSpec), need to mention about it here, in this context it sill be clear why such hierarchy was chosen.

Reply:
I see what you are talking about.
We don't have plan to introduce `topology.cpuManagerPolicy` or `topology.cpuSiblings` now.
So it seems to introduce `v1core.PodSpec.topologyPolicy` is a better way.
(nodeTopologyPolicy, I think it is a little bit wired because we are talking about topology of Pod not a node.)